react-components/intro-to-react-js.md

```diff
@@ -0,0 +1,101 @@
+# Intro to React JS
+
+## Learning Goals
+
+- What is React?
+- Why do we learn React?
+- Who is using React?
+- Intro to the Virtual DOM
+- Intro to Components
```

Comment:
these don't sound very goal-like at the moment. suggest a rewording like the following (feel free to tweak!).

also, I didn't see anything in here about the virtual DOM, so maybe remove that from here? or if this is meant to be more over-arching learning goals for React as a whole, then maybe include some clarification about when that goal will be met.

```suggestion
- Define what React is
- Understand why we learn React
- Discuss who is using React
- Acquire introductory understanding of React Components
```